lib/features/modeling/BpmnUpdater.js

```diff
@@ -346,6 +350,23 @@ BpmnUpdater.prototype.updateBounds = function(shape) {
 
   var di = getDi(shape);
 
+  // adjust internal bounds if they are defined for compatibility reasons
+  if (hasExplicitlyBoundedInternalLabel(shape)) {
+
+    // clear explicit internal bounds on resize
+    if (shape.width !== di.bounds.width || shape.height !== di.bounds.height) {
+      delete di.label.bounds;
```

Comment:
This action is __not__ revertable. Once you resize a sub-process undoing that action won't bring the DI bounds back. Are we aware of that?